src/main/java/Deadlines.java

```diff
@@ -0,0 +1,28 @@
+public class Deadlines extends Task {
```

Comment:
Should this class name be singular instead of plural? Same goes for the other child classes of Task.
```suggestion
public class Deadline extends Task {
```

Reply:
Agreed. I would change the class to singular instead of plural.